designer/server/src/main/scala/pl/touk/nussknacker/ui/process/repository/DeploymentComment.scala

```diff
@@ -54,11 +54,12 @@ case class CommentValidationError(message: String) extends Exception(message)
 
 object CommentValidationError {
   def apply(comment: String, deploymentCommentSettings: DeploymentCommentSettings): CommentValidationError = {
-    deploymentCommentSettings.exampleComment match {
-      case Some(exampleComment) =>
-        new CommentValidationError(s"Bad comment format '$comment'. Example comment: $exampleComment.")
-      case None =>
-        new CommentValidationError(s"Bad comment format '$comment'. Validation pattern: ${deploymentCommentSettings.validationPattern}")
-    }
+    new CommentValidationError(s"Bad comment format '$comment'. " +
+      (deploymentCommentSettings.exampleComment match {
```

Comment:
Let's extract a variable for second explanation part

Reply:
You mean extracting second of concatenated strings?